modules/migration-configuring-cors-4.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-configuring-cors-4_{context}']
+= Configuring cross-origin resource sharing on {product-title} 4 clusters
+
+If you are migrating to or from an {product-title} 4 cluster that does not have the CAM tool installed, you must configure configure cross-origin resource sharing so that the CAM tool can access the cluster's API server.
+
+.Procedure
+
+. Log in to the CAM tool cluster.
+. Obtain the CAM tool URL:
++
+----
+$ oc get -n openshift-migration route/migration -o go-template='(?i}//{{ .spec.host }}(:|\z){{ println }}' | sed 's,\.,\\.,g'
```

Comment:
There is a typo with this '(?i}', it should be '(?i)' as per below in the doc.

Reply:
Will fix. This file is not being used in the current release because we are assuming CAM is installed on target cluster.